pkg/config/config_template.yaml

```diff
@@ -2844,7 +2844,7 @@ api_key:
     ## @param username - string - optional
     ## The username used by devices when sending TRAPs to the Agent.
     #
-    # username: <USERNAME>
+    # - username: <USERNAME>
```

Comment:
`authKey` is part of the same structure as `username`, but is not indented correctly to be part of the same yaml object. I find only one other list of objects in the template (`apm_config.replace_tags`)  and it uses a different style: would it make sense to show the list in the same way here?